flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/SplitAggregateITCase.scala

```diff
@@ -391,6 +391,42 @@ class SplitAggregateITCase(
     val expected = List("1,1,50", "1,ALL,50")
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
+
+  @Test
+  def testDistinctWithRetraction(): Unit = {
```

Comment:
What is this test for?
Without this fix, it will occasional fail?

Reply:
As I said in the description, this PR is cherry picked  from FLINK-10674, including this test. And this test can pass without the changes. I'm also fine to remove this test.

Reply:
We already have `testMinMaxWithRetraction` to test DistinctWithRetraction, so I think we can remove it.